### PR TITLE
JSDK-2770 Mobile Friendly Quickstart Examples

### PR DESCRIPTION
**Pull request by PikaJoyce**
[JIRA Ticket](https://issues.corp.twilio.com/browse/JSDK-2770)

BUG FIX : Remote Reconnection LocalParticipant is not showing video on mobile.

Mobile Friendly Examples UI. This is WIP but I will update as I finish styling!
- [x] Bandwidth Constraints
- [x] Local Video Filter
- [x] Local Video Snapshot
- [x] Media Device Selection
- [x] Codec Preferences
- [x] Share Your Screen
Gif of Screenshare example on Chrome
quickstartUISS
- [x] Dominant Speaker
- [x] Reconnection States and Events 
quickstartUIRECONNC

- [x] Network Quality
- [x] Enabling and Disabling Tracks
- [x] Remote Reconnection States and Events - Done in another PR and is now merged

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: examples/screenshare/public/index.html [hunk 1]
```diff
@@ -5,35 +5,46 @@
...
     <div class="col-sm-6">
       <div class="card">
         <div class="card-block">
           <h4 class="card-title">
             Share Your Screen
           </h4>
-          <pre class="language-javascript"></pre>
+          <button class="btn btn-primary collapsed d-md-none" type="button" data-toggle="collapse" data-target="#collapseSnippet" aria-expanded="false" aria-controls="collapseSnippet">
+            <span class="if-collapsed">Show Snippet</span>
+            <span class="if-not-collapsed">Hide Snippet</span>
```
Comment (manjeshbhargav) [inline examples/screenshare/public/index.html:23]:
```suggestion
            <span class="if-collapsed">Show Code</span>
            <span class="if-not-collapsed">Hide Code</span>
```

Comment (manjeshbhargav) [reply examples/screenshare/public/index.html:23]: @PikaJoyce Can yo umake this change everywhere relevant?

**Comment by PikaJoyce** [comment]
Requested Changes : 
- [ ] ALL EXAMPLES : Add gap between columns so there is small gap between app column and show code button
- [ ] Bandwidth Constraint : Make audio graph append to top left corner of video instead of center.
- [ ] Enable & Disable Tracks : 
       - [ ] Append audio icon to top left of video
       - [ ] BUG FIX : Detach video from DOM, this is buggy.
       - [ ] Rename buttons to enable/disable audio/video
- [ ] Network Quality, Codec Preferences, Media Device Selection, Local Video Filter : The select arrow is no longer visible on mobile emulator.

CHECK. ALL. EXAMPLES. ON. IOS SAFARI + ANDROID CHROME DEVICES.

- [ ] Triple check everything to make sure it is all hunky dory

**Comment by PikaJoyce** [comment]
@manjeshbhargav here's what the new enable disable example looks like! 
quickstartUIenable

**Pull request merged**